pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -522,6 +544,46 @@ func (h *NetworkUtilsHandler) DisableTXOffloadChecksum(ifaceName string) error {
 	return nil
 }
 
+func (h *NetworkUtilsHandler) CreateRADaemon(socketPath string, advertisementIfaceName string, ipv6CIDR string, currentRetry int) error {
+	c, err := net.Dial("unix", socketPath)
+	if err != nil {
+		log.Log.V(4).Warningf("could not connect to the unix domain socket: %v", err)
+
+		if currentRetry > 0 {
```

Comment:
Why do we need a retry mechanism?

Reply:
Seems we stopped needing this shenanigan once we stopped using the additional process.

All the better.